test/lib/meetup_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+require_relative '../support/meetups/sample_api_responses'
+
+class MeetupTest < ActiveSupport::TestCase
+	test "good data is saved" do
+		Meetup.stubs(:get).with("pro/operationcode/groups", options).returns(group_endpoint_response) #Returns response which includes response.parsed_response and response.code
```

Comment:
@leenyburger - the issue is that there is a missing `/` in the url.  Should be equal to this url:

https://github.com/leenyburger/operationcode_backend/blob/43341d615958e2d497153bf08a98a7c2d3239d1d/app/lib/meetup.rb#L19